code/_onclick/hud/new_player.dm

```diff
@@ -348,14 +369,139 @@
 		set_button_status(FALSE)
 		return
 
-/atom/movable/screen/lobby/button/poll/update_overlays()
+/atom/movable/screen/lobby/button/bottom/poll/update_overlays()
 	. = ..()
 	if(new_poll)
 		. += mutable_appearance('icons/hud/lobby/poll_overlay.dmi', "new_poll")
 
-/atom/movable/screen/lobby/button/poll/Click(location, control, params)
+/atom/movable/screen/lobby/button/bottom/poll/Click(location, control, params)
 	. = ..()
 	if(!.)
 		return
 	var/mob/dead/new_player/new_player = hud.mymob
 	new_player.handle_player_polling()
+
+#define SHUTTER_MOVEMENT_DURATION 0.4 SECONDS
+#define SHUTTER_WAIT_DURATION 0.2 SECONDS
+
+/atom/movable/screen/lobby/button/collapse
+	name = "Collapse Lobby Menu"
+	icon = 'icons/hud/lobby/collapse_expand.dmi'
+	icon_state = "collapse"
+	base_icon_state = "collapse"
+	layer = LOBBY_COLLAPSE_BUTTON_LAYER
+	screen_loc = "TOP:-82,CENTER:-54"
+	always_shown = TRUE
+	var/matrix/ce_button_matrix = matrix()
+
+/atom/movable/screen/lobby/button/collapse/Click(location, control, params)
+	. = ..()
+	if(!.)
+		return
+	var/mob/dead/new_player/new_player = hud.mymob
+	//whether we are currently showing the menu to the client or not
+	var/inventory_status = hud.inventory_shown
```

Comment:
yeah ok why are we using inventory_shown for this

Reply:
that var exists on parent /hud/ and doesn't get used otherwise iirc, so i used it here

Reply:
why not make your own var for this?

Reply:
less fiddling with mob's `hud_used` or hud elements' `hud` as those vars are /datum/hud/ and would have to be typechecked into our /new_player subtype before accessing the new var. so i decided why not use the one that's on the parent, isn't used on parent (iirc, i saw like one usage of that in the carbon hud for equipmemt slots) and would basically behave the same

Reply:
I don't like using it like this, it's very obviously not what it's for and it leads to comments that should be redundant like this one

Reply:
this could at best overlap with changing the hud's mode with F12 (which is funnily enough this entire PR but without animation), as toggling the hud to normal mode while the hud is collapsed may re-enable the hidden menu buttons. 

i'll make a var for this to avoid that but other than that it woulda been a right fit actually i think

Reply:
https://github.com/tgstation/tgstation/pull/76443/commits/e74765b3ea02e603b27f346dc73142c28b13f527